lib/fog/compute/vmfusion.rb

```diff
@@ -1,8 +1,8 @@
 module Fog
   module Compute
     class Vmfusion < Fog::Service
-      autoload :Server, 'fog/compute/vmfusion/server'
-      autoload :Servers, 'fog/compute/vmfusion/servers'
+      autoload :Server, File.expand_path('../vmfusion/server', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.